Sources/Controllers/MessagesViewController.swift

```diff
@@ -82,6 +82,12 @@ open class MessagesViewController: UIViewController {
         setupDelegates()
         addMenuControllerObservers()
     }
+    
+    open override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        view.layoutIfNeeded()
+        messagesCollectionView.collectionViewLayout.invalidateLayout()
```

Comment:
The reason I haven't merged this is I'm a little concerned that the fix involves invalidating the layout on `viewWillAppear`. I have to check this one

Reply:
👌